riyac4/riyacherlakola.com
# Image Copy Feature

- Compress image if it JPEG
- Resize (Max with 1024) if it's larger than 1024

## How to test

1. Delete some files in `dist/articles/blog-images folder`
2. run `python src/site-builder.py`
3. check the timestamp of newly generated files. Existing files should not be replaced.
4. for jpg files, make sure it's compressed and resized
